pipestat/pipestat.py

```diff
@@ -383,11 +404,32 @@ def initialize_filebackend(self, record_identifier, results_file_path, flag_file
 
         return
 
+    def initialize_pephubbackend(self, record_identifier: str = None, pephub_path: str = None):
+        """
+        Initializes the pephub backend
+        :param str record_identifier: the record identifier
+        :param str pephub_path: the path to the pephub registry
+        """
+        self.backend = PEPHUBBACKEND(
+            record_identifier,
+            pephub_path,
+            self.cfg[PIPELINE_NAME],
+            self.cfg[PIPELINE_TYPE],
+            self.cfg[SCHEMA_KEY],
+            self.cfg[STATUS_SCHEMA_KEY],
+            self.cfg[RESULT_FORMATTER],
+        )
+
     @check_dependencies(
         dependency_list=["DBBackend"],
         msg="Missing required dependencies for this usage, e.g. try pip install pipestat['dbbackend']",
     )
-    def initialize_dbbackend(self, record_identifier, show_db_logs):
+    def initialize_dbbackend(self, record_identifier: str = None, show_db_logs: bool = False):
```

Comment:
Does it return anything?

Reply:
No. None of the backend initialization functions return anything.